pkgs/by-name/co/container-diff/package.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, buildGoModule
+, installShellFiles
+, testers
+, container-diff
+}:
+
+buildGoModule rec {
+  pname = "container-diff";
+  version = "0.19.0";
+
+  src = fetchFromGitHub {
+    owner = "GoogleContainerTools";
+    repo = "container-diff";
+    rev = "v${version}";
+    hash = "sha256-oihyH4anPDV3vMna7ZKg+5cn9QQVgrUw4qUi2qcyv0w=";
+  };
+
+  vendorHash = "sha256-aHF1y8/aoXQ73TwSPymj4yngfeyMB48u6/2Kz4JV/qk=";
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/GoogleContainerTools/container-diff/pkg/version.version=${version}"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  doCheck = false; # uses network
```

Comment:
Are there particular tests that use the network - could some be run with a subset disabled used `disabledTests`?